spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -356,12 +356,13 @@ spring.cloud.deployer.cloudfoundry.task.apiTimeout=360
 ```
 
 Note that you can set the following properties `spring.cloud.deployer.cloudfoundry.services`,
-`spring.cloud.deployer.cloudfoundry.memory`, and `spring.cloud.deployer.cloudfoundry.disk` as part of an individual
-deployment request prefixed by the `app.<name of application>`.  For example
+`spring.cloud.deployer.cloudfoundry.buildpack` or the Spring Cloud Deployer standard
```

Comment:
Those properties are mentioned because they're supported. This is our way of letting users know that they exist. Some of them have examples, some of them don't